ext/page_log/palm/palm.c

```diff
@@ -1241,7 +1248,9 @@ palm_terminate(WT_PAGE_LOG *storage, WT_SESSION *session)
     ret = 0;
     palm = (PALM *)storage;
 
-    if (--palm->reference_count != 0)
+    uint32_t new_ref_count = atomic_fetch_sub(&palm->reference_count, 1);
+    /* The last reference is 1. */
```

Comment:
This comment doesn't make sense to me -- doesn't "1" imply it's the last reference already?